packages/arb-token-bridge-ui/src/hooks/useEmbedMode.ts

```diff
@@ -0,0 +1,10 @@
+import { useArbQueryParams, ModeParamEnum } from './useArbQueryParams'
+
+export function useEmbedMode() {
+  const [{ mode }] = useArbQueryParams()
+
+  const embedMode = mode === ModeParamEnum.EMBED
+  // later, we can have other booleans like `embedCompactMode` depending on the requirements
+
+  return { mode, embedMode }
```

Comment:
I find it odd that `useEmbedMode` also returns mode enum. I think we should have one of the following:
1. useIsEmbedMode -> boolean & useIsCompactMode -> boolean
2. useMode -> ModeParamEnum